projects/twitter/build_dict.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
shouldnt the eval_ppl script do this thing by default or? 
and we send it some params maybe..

Reply:
I'm not sure what you're asking

Reply:
put build_dict in model zoo instead?

Reply:
put in zoo instead?  i think it's nicer